.idea/confidence-openfeature-provider-go.iml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did you mean to commit the `.idea` folder?

Reply:
nope. thanks.